app/views/dashboards/_budget_bar.html.erb

```diff
@@ -1,14 +1,29 @@
-<div id="overview" class="margin-bottom">
-  <h1 class="border-bottom title">Week of <%= week_range %></h1>
+<div class="col-sm-12" id='budget-bar'>
+  <div class="row">
+    <div class="progress" id='budget-progress-bar'>
```

Comment:
May be worth trying the aria-value stuff as detailed here: https://getbootstrap.com/docs/3.3/components/#progress-alternatives Way simpler than a bunch of obscured helper methods.